h/templates/app.html.jinja2

```diff
@@ -49,6 +50,10 @@
                 on-close="accountDialog.visible=false"></login-form>
       <sidebar-tutorial ng-if="isSidebar"></sidebar-tutorial>
       <share-dialog ng-show="shareDialog.visible"></share-dialog>
+      <about-this-version-dialog ng-show="aboutThisVersionDialog.visible"
```

Comment:
The `<share-dialog>` above uses `ng-show` but this is fixed in https://github.com/hypothesis/h/pull/3215